### ISSUE TITLE
Keyerror in na_ontap_dns after 22.8.0 upgrade

### ISSUE THREAD

**Issue by peteshaped** [ansible-collections/netapp.ontap#185]
### Summary

When I use the following resource:

```- name: Setup DNS
  na_ontap_dns:
    state: present
    vserver: "{{ inventory_hostname }}"
    domains: "{{ item.dns_domains }}"
    nameservers: "{{ item.dns_nameservers }}"
    skip_validation: "{{ netapp_dns_skip_validation }}"
    hostname: "{{ netapp_hostname }}"
    username: "{{ netapp_username }}"
    password: "{{ netapp_password }}"
    https: "{{ netapp_use_https }}"
    validate_certs: "{{ netapp_validate_certs }}"
    use_rest: "{{ netapp_use_rest }}"
  delegate_to: localhost
  with_items:
    "{{ netapp_dns }}"
  when: netapp_dns != None
```

Previously in 22.7.0 I would see the following:

```
$ ansible-galaxy collection install -r requirements.yaml -U
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Downloading https://galaxy.ansible.com/api/v3/plugin/ansible/content/published/collections/artifacts/netapp-ontap-22.7.0.tar.gz to /home/username/.ansible/tmp/ansible-local-9526531api7xg/tmprq_tvdmp/netapp-ontap-22.7.0-823navb2
Installing 'netapp.ontap:22.7.0' to '/home/username/.ansible/collections/ansible_collections/netapp/ontap'
netapp.ontap:22.7.0 was installed successfully
$ ansible-playbook -vvv playbooks/main.yaml
<snip>
TASK [netapp_cluster_setup : Setup DNS] ****************************************************************************************************************************************************************************************************************************************************************************task path: /home/username/wslcode/it-storage-ansible/roles/netapp_cluster_setup/tasks/main.yml:16
redirecting (type: modules) ansible.builtin.na_ontap_dns to netapp.ontap.na_ontap_dns
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: username
<localhost> EXEC /bin/sh -c 'echo ~username && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/username/.ansible/tmp `"&& mkdir "` echo /home/username/.ansible/tmp/ansible-tmp-1698955915.2987483-95436-113327368795690 `" && echo ansible-tmp-1698955915.2987483-95436-113327368795690="` echo /home/username/.ansible/tmp/ansible-tmp-1698955915.2987483-95436-113327368795690 `" ) && sleep 0'
redirecting (type: modules) ansible.builtin.na_ontap_dns to netapp.ontap.na_ontap_dns
Using module file /home/username/.ansible/collections/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py
<localhost> PUT /home/username/.ansible/tmp/ansible-local-95352bi7lnnmn/tmp8i2qkox5 TO /home/username/.ansible/tmp/ansible-tmp-1698955915.2987483-95436-113327368795690/AnsiballZ_na_ontap_dns.py
<localhost> EXEC /bin/sh -c 'chmod u+x /home/username/.ansible/tmp/ansible-tmp-1698955915.2987483-95436-113327368795690/ /home/username/.ansible/tmp/ansible-tmp-1698955915.2987483-95436-113327368795690/AnsiballZ_na_ontap_dns.py && sleep 0'
<localhost> EXEC /bin/sh -c '/home/username/.pyenv/versions/3.11.4/bin/python3.11 /home/username/.ansible/tmp/ansible-tmp-1698955915.2987483-95436-113327368795690/AnsiballZ_na_ontap_dns.py && sleep 0'
<localhost> EXEC /bin/sh -c 'rm -f -r /home/username/.ansible/tmp/ansible-tmp-1698955915.2987483-95436-113327368795690/ > /dev/null 2>&1 && sleep 0'
ok: [netapp02 -> localhost] => (item={'dns_domains': 'localdomain.local', 'dns_nameservers': '192.168.0.85,192.168.0.86'}) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "cert_filepath": null,
            "domains": [
                "localdomain.local"
            ],
            "feature_flags": null,
            "force_ontap_version": null,
            "hostname": "netapp02.it.localdomain.local",
            "http_port": null,
            "https": true,
            "key_filepath": null,
            "nameservers": [
                "192.168.0.85",
                "192.168.0.86"
            ],
            "ontapi": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "skip_validation": false,
            "state": "present",
            "use_rest": "always",
            "username": "admin",
            "validate_certs": false,
            "vserver": "netapp02"
        }
    },
    "item": {
        "dns_domains": "localdomain.local",
        "dns_nameservers": "192.168.0.85,192.168.0.86"
    }
}
<snip>
```

Since upgrading to 22.8.0, I see a uuid key error:

```
$ ansible-galaxy collection install -r requirements.yaml -U
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Downloading https://galaxy.ansible.com/api/v3/plugin/ansible/content/published/collections/artifacts/netapp-ontap-22.8.0.tar.gz to /home/username/.ansible/tmp/ansible-local-96152mt14q_do/tmpieyqxqmx/netapp-ontap-22.8.0-2j_7zc98
Installing 'netapp.ontap:22.8.0' to '/home/username/.ansible/collections/ansible_collections/netapp/ontap'
netapp.ontap:22.8.0 was installed successfully
$ ansible-playbook -vvv playbooks/main.yaml
<SNIP>
TASK [netapp_cluster_setup : Setup DNS] ****************************************************************************************************************************************************************************************************************************************************************************task path: /home/username/wslcode/it-storage-ansible/roles/netapp_cluster_setup/tasks/main.yml:16
redirecting (type: modules) ansible.builtin.na_ontap_dns to netapp.ontap.na_ontap_dns
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: username
<localhost> EXEC /bin/sh -c 'echo ~username && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/username/.ansible/tmp `"&& mkdir "` echo /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955 `" && echo ansible-tmp-1698956126.8424404-96329-120714306185955="` echo /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955 `" ) && sleep 0'
redirecting (type: modules) ansible.builtin.na_ontap_dns to netapp.ontap.na_ontap_dns
Using module file /home/username/.ansible/collections/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py
<localhost> PUT /home/username/.ansible/tmp/ansible-local-962452evc6ln0/tmpo7ddoc6m TO /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py
<localhost> EXEC /bin/sh -c 'chmod u+x /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/ /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py && sleep 0'
<localhost> EXEC /bin/sh -c '/home/username/.pyenv/versions/3.11.4/bin/python3.11 /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py && sleep 0'
<localhost> EXEC /bin/sh -c 'rm -f -r /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py", line 107, in <module>
    _ansiballz_main()
  File "/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py", line 99, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py", line 47, in invoke_module
    runpy.run_module(mod_name='ansible_collections.netapp.ontap.plugins.modules.na_ontap_dns', init_globals=dict(_module_fqn='ansible_collections.netapp.ontap.plugins.modules.na_ontap_dns', _modlib_path=modlib_path),
  File "<frozen runpy>", line 226, in run_module
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 368, in <module>
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 364, in main
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 343, in apply
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 274, in get_dns
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 265, in get_dns_rest
KeyError: 'uuid'
failed: [netapp02 -> localhost] (item={'dns_domains': 'localdomain.local', 'dns_nameservers': '192.168.0.85,192.168.0.86'}) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": {
        "dns_domains": "localdomain.local",
        "dns_nameservers": "192.168.0.85,192.168.0.86"
    },
    "module_stderr": "Traceback (most recent call last):\n  File \"/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.netapp.ontap.plugins.modules.na_ontap_dns', init_globals=dict(_module_fqn='ansible_collections.netapp.ontap.plugins.modules.na_ontap_dns', _modlib_path=modlib_path),\n  File \"<frozen runpy>\", line 226, in run_module\n  File \"<frozen runpy>\", line 98, in _run_module_code\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 368, in <module>\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 364, in main\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 343, in apply\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 274, in get_dns\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 265, in get_dns_rest\nKeyError: 'uuid'\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}
<SNIP>
```

### Component Name

na_ontap_dns

### Ansible Version

```console
$ ansible --version
ansible [core 2.15.5]
  config file = /home/username/wslcode/it-storage-ansible/ansible.cfg
  configured module search path = ['/home/username/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/username/.pyenv/versions/3.11.4/lib/python3.11/site-packages/ansible
  ansible collection location = /home/username/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/username/.pyenv/versions/3.11.4/bin/ansible
  python version = 3.11.4 (main, Oct 26 2023, 21:31:41) [GCC 11.4.0] (/home/username/.pyenv/versions/3.11.4/bin/python3.11)
  jinja version = 3.1.2
  libyaml = True
```

### ONTAP Collection Version

```console
$ ansible-galaxy collection list
$ ansible-galaxy collection list

# /home/username/.ansible/collections/ansible_collections
Collection                    Version
----------------------------- -------
netapp.ontap                  22.8.0

# /home/username/.pyenv/versions/3.11.4/lib/python3.11/site-packages/ansible_collections
Collection                    Version
----------------------------- -------
amazon.aws                    6.5.0
ansible.netcommon             5.2.0
ansible.posix                 1.5.4
ansible.utils                 2.11.0
ansible.windows               1.14.0
arista.eos                    6.1.2
awx.awx                       22.7.0
azure.azcollection            1.18.1
check_point.mgmt              5.1.1
chocolatey.chocolatey         1.5.1
cisco.aci                     2.7.0
cisco.asa                     4.0.2
cisco.dnac                    6.7.5
cisco.intersight              1.0.27
cisco.ios                     4.6.1
cisco.iosxr                   5.0.3
cisco.ise                     2.5.16
cisco.meraki                  2.16.5
cisco.mso                     2.5.0
cisco.nso                     1.0.3
cisco.nxos                    4.4.0
cisco.ucs                     1.10.0
cloud.common                  2.1.4
cloudscale_ch.cloud           2.3.1
community.aws                 6.3.0
community.azure               2.0.0
community.ciscosmb            1.0.6
community.crypto              2.15.1
community.digitalocean        1.24.0
community.dns                 2.6.2
community.docker              3.4.9
community.fortios             1.0.0
community.general             7.5.0
community.google              1.0.0
community.grafana             1.5.4
community.hashi_vault         5.0.0
community.hrobot              1.8.1
community.libvirt             1.3.0
community.mongodb             1.6.3
community.mysql               3.7.2
community.network             5.0.0
community.okd                 2.3.0
community.postgresql          2.4.3
community.proxysql            1.5.1
community.rabbitmq            1.2.3
community.routeros            2.10.0
community.sap                 1.0.0
community.sap_libs            1.4.1
community.skydive             1.0.0
community.sops                1.6.6
community.vmware              3.10.0
community.windows             1.13.0
community.zabbix              2.1.0
containers.podman             1.10.3
cyberark.conjur               1.2.2
cyberark.pas                  1.0.23
dellemc.enterprise_sonic      2.2.0
dellemc.openmanage            7.6.1
dellemc.powerflex             1.9.0
dellemc.unity                 1.7.1
f5networks.f5_modules         1.26.0
fortinet.fortimanager         2.2.1
fortinet.fortios              2.3.2
frr.frr                       2.0.2
gluster.gluster               1.0.2
google.cloud                  1.2.0
grafana.grafana               2.2.3
hetzner.hcloud                1.16.0
hpe.nimble                    1.1.4
ibm.qradar                    2.1.0
ibm.spectrum_virtualize       1.12.0
infinidat.infinibox           1.3.12
infoblox.nios_modules         1.5.0
inspur.ispim                  1.3.0
inspur.sm                     2.3.0
junipernetworks.junos         5.3.0
kubernetes.core               2.4.0
lowlydba.sqlserver            2.2.1
microsoft.ad                  1.3.0
netapp.aws                    21.7.0
netapp.azure                  21.10.0
netapp.cloudmanager           21.22.0
netapp.elementsw              21.7.0
netapp.ontap                  22.7.0
netapp.storagegrid            21.11.1
netapp.um_info                21.8.0
netapp_eseries.santricity     1.4.0
netbox.netbox                 3.14.0
ngine_io.cloudstack           2.3.0
ngine_io.exoscale             1.1.0
ngine_io.vultr                1.1.3
openstack.cloud               2.1.0
openvswitch.openvswitch       2.1.1
ovirt.ovirt                   3.2.0
purestorage.flasharray        1.21.0
purestorage.flashblade        1.14.0
purestorage.fusion            1.6.0
sensu.sensu_go                1.14.0
servicenow.servicenow         1.0.6
splunk.es                     2.1.0
t_systems_mms.icinga_director 1.33.1
telekom_mms.icinga_director   1.34.1
theforeman.foreman            3.14.0
vmware.vmware_rest            2.3.1
vultr.cloud                   1.10.0
vyos.vyos                     4.1.0
wti.remote                    1.0.5
```

### ONTAP Version

```console
netapp02::> version
NetApp Release 9.10.1P12: Thu Apr 13 00:30:59 UTC 2023
```

### Playbook

```console
- name: Setup DNS
  na_ontap_dns:
    state: present
    vserver: "{{ inventory_hostname }}"
    domains: "{{ item.dns_domains }}"
    nameservers: "{{ item.dns_nameservers }}"
    skip_validation: "{{ netapp_dns_skip_validation }}"
    hostname: "{{ netapp_hostname }}"
    username: "{{ netapp_username }}"
    password: "{{ netapp_password }}"
    https: "{{ netapp_use_https }}"
    validate_certs: "{{ netapp_validate_certs }}"
    use_rest: "{{ netapp_use_rest }}"
  delegate_to: localhost
  with_items:
    "{{ netapp_dns }}"
  when: netapp_dns != None
```

### Steps to Reproduce

```yaml (paste below)
ansible-playbook -vvv playbooks/main.yaml
```

### Expected Results

I expected it to validate the DNS config was correct (as no changes were expected to be made from ansible vs actual config)

### Actual Results

```console
TASK [netapp_cluster_setup : Setup DNS] ****************************************************************************************************************************************************************************************************************************************************************************task path: /home/username/wslcode/it-storage-ansible/roles/netapp_cluster_setup/tasks/main.yml:16
redirecting (type: modules) ansible.builtin.na_ontap_dns to netapp.ontap.na_ontap_dns
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: username
<localhost> EXEC /bin/sh -c 'echo ~username && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/username/.ansible/tmp `"&& mkdir "` echo /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955 `" && echo ansible-tmp-1698956126.8424404-96329-120714306185955="` echo /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955 `" ) && sleep 0'
redirecting (type: modules) ansible.builtin.na_ontap_dns to netapp.ontap.na_ontap_dns
Using module file /home/username/.ansible/collections/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py
<localhost> PUT /home/username/.ansible/tmp/ansible-local-962452evc6ln0/tmpo7ddoc6m TO /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py
<localhost> EXEC /bin/sh -c 'chmod u+x /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/ /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py && sleep 0'
<localhost> EXEC /bin/sh -c '/home/username/.pyenv/versions/3.11.4/bin/python3.11 /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py && sleep 0'
<localhost> EXEC /bin/sh -c 'rm -f -r /home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py", line 107, in <module>
    _ansiballz_main()
  File "/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py", line 99, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py", line 47, in invoke_module
    runpy.run_module(mod_name='ansible_collections.netapp.ontap.plugins.modules.na_ontap_dns', init_globals=dict(_module_fqn='ansible_collections.netapp.ontap.plugins.modules.na_ontap_dns', _modlib_path=modlib_path),
  File "<frozen runpy>", line 226, in run_module
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 368, in <module>
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 364, in main
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 343, in apply
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 274, in get_dns
  File "/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py", line 265, in get_dns_rest
KeyError: 'uuid'
failed: [netapp02 -> localhost] (item={'dns_domains': 'localdomain.local', 'dns_nameservers': '192.168.0.85,192.168.0.86'}) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": {
        "dns_domains": "localdomain.local",
        "dns_nameservers": "192.168.0.85,192.168.0.86"
    },
    "module_stderr": "Traceback (most recent call last):\n  File \"/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/username/.ansible/tmp/ansible-tmp-1698956126.8424404-96329-120714306185955/AnsiballZ_na_ontap_dns.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.netapp.ontap.plugins.modules.na_ontap_dns', init_globals=dict(_module_fqn='ansible_collections.netapp.ontap.plugins.modules.na_ontap_dns', _modlib_path=modlib_path),\n  File \"<frozen runpy>\", line 226, in run_module\n  File \"<frozen runpy>\", line 98, in _run_module_code\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 368, in <module>\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 364, in main\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 343, in apply\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 274, in get_dns\n  File \"/tmp/ansible_na_ontap_dns_payload_msoeewrl/ansible_na_ontap_dns_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_dns.py\", line 265, in get_dns_rest\nKeyError: 'uuid'\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}
```

**Comment by carchi8py** [comment]
This is a bug we have DEVOPS-6520 story open for this and will fix this in a bug patch.

**Comment by carchi8py** [comment]
This should be fixed in 22.8.1

**Comment by ansibleguy76** [comment]
any idea when 22.8.1 will be released.
